components/Workspaces/TrackedRepoWizard/SelectOrgReposStep.tsx

```diff
@@ -78,11 +83,19 @@ export const SelectOrgReposStep = ({
 
   return (
     <div className="flex flex-col gap-4 h-96 max-h-96">
-      <div className="flex items-center gap-2 font-semibold">
-        <Avatar contributor={organization} size="xsmall" />
-        <span>{organization}</span>
+      <div className="flex w-full justify-between gap-2 font-semibold">
+        <div className="flex gap-2 justify-start align-middle">
+          <Avatar contributor={organization} size="small" />
+          <span className="flex align-middle">{organization}</span>
+        </div>
+        <div className="flex justify-end">
+          <DayRangePicker onDayRangeChanged={(e) => setRange(Number(e))} />
+        </div>
       </div>
-      <p>Select the organization repositories that you want to track.</p>
+      <p>
+        Select the organization repositories updated in the last <span className="font-bold">{range} days</span> that
+        you want to track.
```

Comment:
Thinking of an edge case here: is there now no way to import all repos from an org that go beyond being updated in 1 year? I.e., all repos regardless of a filter. For example, with the OpenSauced org, we end up missing all those old repos that we haven't touched in some time.

Which, if this is the intend, then that makes sense 👍🏼

Reply:
Yes, that's correct. Feedback given to @bdougie was that older repos that haven't been updated shouldn't be listed on the org import. I suppose they could go back and copy/paste the other ones in